### PR TITLE
Adds lighting to lambertDiffuseMultiplier spec

### PR DESCRIPTION
**Pull request by sanjeetsuhag**
This fixes https://github.com/CesiumGS/cesium/issues/10258 and https://github.com/CesiumGS/cesium/issues/10168

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @sanjeetsuhag!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/lambert-test-fix/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Comment by lilleyse** [comment]
This doesn't seem like the right fix. The test should work even if atmosphere is disabled.

**Comment by lilleyse** [comment]
Could you try setting the time of day as suggested in https://github.com/CesiumGS/cesium/issues/10258? Depending on the light direction the lamber diffuse multiplier will have no effect.

```
float diffuseIntensity = clamp(czm_getLambertDiffuse(czm_lightDirectionEC, normalize(v_normalEC)) * u_lambertDiffuseMultiplier + 0.3, 0.0, 1.0);
```

**Comment by sanjeetsuhag** [comment]
@lilleyse Thanks for the suggestion, committed the fix.

File: Specs/Scene/GlobeSpec.js [hunk 1]
```diff
@@ -315,7 +316,13 @@ describe(
     });
 
     it("renders terrain with lambertDiffuseMultiplier", function () {
+      const renderOptions = {
+        scene: scene,
+        time: new JulianDate(2557522.0),
+      };
       globe.enableLighting = true;
+      globe.dynamicAtmosphereLighting = true;
+      globe.dynamicAtmosphereLightingFromSun = true;
```
Comment (lilleyse) [inline Specs/Scene/GlobeSpec.js:325]: Can these be removed now?

File: Specs/Scene/GlobeSpec.js [hunk 2]
```diff
@@ -315,7 +316,13 @@ describe(
     });
 
     it("renders terrain with lambertDiffuseMultiplier", function () {
+      const renderOptions = {
+        scene: scene,
+        time: new JulianDate(2557522.0),
+      };
```
Comment (lilleyse) [inline Specs/Scene/GlobeSpec.js:322]: Could you also do this in the test above? It's written very similarly.

**Comment by sanjeetsuhag** [comment]
Committed the fixes.

So, from the syntax of these, what I'm guessing is that you only use `toRenderAndCall` when you want to read back the RGBA value. Otherwise, just use `expect().toRender` or `expect().notToRender`.

**Pull request merged**